specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1121 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HDInsightJobManagementClient",
```

Comment:
>HDInsightJobManagementClient [](start = 14, length = 28)

After talked with Dongwei, we need to change it to JobClient.

Reply:
Updated.